web/src/index.html

```diff
@@ -10,5 +10,11 @@
   <body>
     <div id="root"></div>
     <script type="module" src="./index.js"></script>
+    <!--
+      The Cockpit server returns "po.<LANG>.js" content for the "po.js" request,
+      the requested language is obtained from the "CockpitLang" cookie sent in
+      the HTTP header. For missing translations it returns an empty string.
```

Comment:
> For missing translations it returns an empty string.

Does this mean that if I miss to translate some strings, there will be empty texts?

```suggestion
      the HTTP header. For missing translations it returns an empty script.
```